scripts/script.yaml

```diff
@@ -352,48 +369,35 @@
       
       - name: exposures
         steps:
-          - say: מה מספר האנשים השונים (לא כולל בני ביתך) ששהית בקרבתם מעל ל-5 דקות ביממה האחרונה?
-          - wait:
-              variable: exposure_met_people
-              placehoder: מספר האנשים
-              validation: "[0-9]+"
-              input-kind: number
-
-          - say: כמה בני משפחה או דיירים אחרים מתגוררים איתך באותו הבית או באותה הדירה?
-          - wait:
-              variable: flatmates
-              placehoder: מספר האנשים
-              validation: "[0-9]+"
-              input-kind: number
-          
-          - say: כמה מהם בני 70 או יותר?
+          - say: מה מספר האנשים השונים ששהית בקרבתם מעל ל-15 דקות בטווח 2 מטרים ביממה האחרונה?
```

Comment:
This is a question with no possibility answer. Feels like a bug.

Reply:
@akariv tried to clarify,  I believe @mushon will be able to improve the wording here on staging